### ISSUE TITLE
Does this actually work on HKS? Havok Script

### ISSUE THREAD

**Issue by MagnificentS** [ludkiller/HKSDecompiler#1]
So I was interested in using this for some Darksouls modding
But I couldn't get it to work.

**Comment by Convery** [comment]
No, it was a proof of concept. Master131, Ghost, or Kokole may have proper
implementations.

On 13 Aug 2018 04:57, "MagnificentS" <notifications@github.com> wrote:

So I was interested in using this for some Darksouls modding
But I couldn't get it to work.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/pareshchouhan/HKSDecompiler/issues/1>, or mute the
thread

**Comment by MagnificentS** [comment]
Any Ideas how I may contact them (Aka an actual forum/reddit) as the github handles dont seem to be them
